src/Xamarin.Android.Build.Tasks/Utilities/MonoAndroidHelper.Linker.cs

```diff
@@ -18,12 +18,6 @@ public partial class MonoAndroidHelper
 			"Mono.Data.Sqlite.dll",
 			"Mono.Posix.dll",
 		};
-		internal static readonly HashSet<string> FrameworkAssembliesToTreatAsUserAssemblies = new HashSet<string> (StringComparer.OrdinalIgnoreCase) {
-			"OpenTK-1.0.dll",
-			"Mono.Android.Support.v13.dll",
-			"Mono.Android.Support.v4.dll",
-			"Xamarin.Android.NUnitLite.dll",
```

Comment:
Don't we still use NUnitLite?

Reply:
This code looks like it has a list of assembly names it treats as "framework assemblies". Doesn't seem like `Xamarin.Android.NUnitLite` should be counted as one if it were found.

We don't ship this assembly in the Microsoft.Android.Runtime packs, those only contain `Mono.Android.dll`, `Mono.Android.Export.dll`, and `Java.Interop.dll`.